Shared/Samples/Create mobile geodatabase/CreateMobileGeodatabaseView.swift

```diff
@@ -0,0 +1,137 @@
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The point on the map where the user tapped.
+    @State private var tapMapPoint = Point(x: 0, y: 0)
+    
+    /// A Boolean indicating whether the feature table sheet is showing.
+    @State private var isShowingTableSheet = false
+    
+    var body: some View {
+        MapView(map: model.map)
+            .onSingleTapGesture { _, mapPoint in
+                tapMapPoint = mapPoint
+            }
+            .task {
+                // Create the initial geodatabase.
+                await model.createGeodatabase()
+            }
+            .task(id: tapMapPoint) {
+                // Add a feature at the tapped map point.
+                await model.addFeature(at: tapMapPoint)
+            }
+            .overlay(alignment: .top) {
+                Text("Number of features added: \(model.features.count)")
+                    .frame(maxWidth: .infinity, alignment: .center)
+                    .padding(8)
+                    .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+            }
+            .toolbar {
+                ToolbarItemGroup(placement: .bottomBar) {
+                    tableButton
+                        .disabled(model.features.isEmpty)
+                    
+                    Button {
+                        model.presentShareSheet()
+                    } label: {
+                        Image(systemName: "square.and.arrow.up")
+                    }
+                }
+            }
+            .alert(isPresented: $model.isShowingErrorAlert, presentingError: model.error)
+    }
+}
+
+private extension CreateMobileGeodatabaseView {
+    /// The button that brings up the feature table sheet.
+    @ViewBuilder var tableButton: some View {
+        /// The button to bring up the sheet.
+        let button = Button("View Table") {
+            isShowingTableSheet = true
+        }
+        
+        if #available(iOS 16, *) {
+            button
+                .popover(isPresented: $isShowingTableSheet, arrowEdge: .bottom) {
+                    tableList
+                        .presentationDetents([.fraction(0.5)])
+#if targetEnvironment(macCatalyst)
+                        .frame(minWidth: 300, minHeight: 270)
+#else
+                        .frame(minWidth: 320, minHeight: 390)
+#endif
+                }
+        } else {
+            button
+                .sheet(isPresented: $isShowingTableSheet) {
+                    tableList
+                }
+        }
+    }
+    
+    /// The list of features in the feature table.
+    var tableList: some View {
+        NavigationView {
+            List {
+                Section("OID and Collection Timestamp") {
+                    ForEach(model.features, id: \.self) { feature in
+                        HStack {
+                            Text(String(feature.oid))
+                            Spacer()
```

Comment:
My personal preference would be to have the variable white space at the end. This way the text is closer to the number and the leading edges of the date/time strings are aligned. (You can see below, when using `Spacer()`, #14 is offset because of the "1" in the time) But it's just personal preference, not a big deal.

```suggestion
                                .padding(.trailing, 8)
```
![image](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/3998072/bb8833bf-202b-4346-a68e-3d1c03495c80)

Reply:
With `padding`: